### PR TITLE
Feature/update grafana fix 122

### PR DESCRIPTION
**Pull request by smerlos**
Closing this task request as completed.

Reference:
- Task Request #122 

# [](https://github.com/sighupio/fury-kubernetes-monitoring/compare/v2.1.0-rc0...v) (2023-02-07)

### Bug Fixes

* **grafana:** update annotations reference to 9.3.6 ([af961f7](https://github.com/sighupio/fury-kubernetes-monitoring/commit/af961f7458d2a2c2950f0d880ababc202589a32e))

### Features

* **image:** update grafana and sidecard image ([bb0e106](https://github.com/sighupio/fury-kubernetes-monitoring/commit/bb0e106ea7f93aab7a173b8d9ed7a62a9d93137a))

### Documentation

* **grafana:** update README.md with the new version ([e874b98](https://github.com/sighupio/fury-kubernetes-monitoring/commit/e874b98f1fd15adb58c9d75971483e4fbf705e43))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Kubernetes Fury Monitoring provides the following packages:
 | [prometheus-operated](katalog/prometheus-operated)     | `2.36.1` | Prometheus instance deployed with Prometheus Operator's CRD                                                 |
 | [alertmanager-operated](katalog/alertmanager-operated) | `0.24.0` | Alertmanager instance deployed with Prometheus Operator's CRD                                               |
 | [blackbox-exporter](katalog/blackbox-exporter)         | `0.24.0` | Prometheus exporter that allows blackbox probing of endpoints over HTTP, HTTPS, DNS, TCP, ICMP and gRPC.    |
-| [grafana](katalog/grafana)                             | `8.5.5`  | Grafana deployment to query and visualize metrics collected by Prometheus                                   |
+| [grafana](katalog/grafana)                             | `9.3.6`  | Grafana deployment to query and visualize metrics collected by Prometheus                                   |
```
Comment (nutellinoit) [inline README.md:53]: Let's leave this info on the main README as the previous one for now, we will update the main README when we create the release note for the new version

**Review by nutellinoit — CHANGES_REQUESTED** [review]
LGTM, with one minor change

**Comment by nutellinoit** [comment]
Also, is 9.3.6 the same version used in Kube prometheus upstream project?

File: katalog/grafana/config.yaml [hunk 2]
```diff
@@ -1,15 +1,11 @@
-# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
```
Comment (nutellinoit) [inline katalog/grafana/config.yaml:1]: Copyright is missing on all the files, you need to maintain that

File: README.md (hunk 1 above)
Comment (ralgozino) [reply README.md:53]: IMHO is OK to update the readme as part of this PR. we usually do it all at once, but now that we are updating piece by piece we can update the readme with each PR. So the document reflects the status of the branch.

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Check the PR, the grafana upstream version seems to be 9.3.2 but you are using 9.3.6 on the main kustomization file.

I also see a new dashboard nodes-darwin but i don't think we need it (is it for macos nodes? 👀 )

you also changed files on other packages, check that and revert the changes

File: katalog/grafana/prometheusRule.yaml [hunk 3]
```diff
@@ -1,15 +1,11 @@
-# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
-# Use of this source code is governed by a BSD-style
-# license that can be found in the LICENSE file.
-
```
Comment (ralgozino) [inline katalog/grafana/prometheusRule.yaml:4]: please re-add the copyright header

File: katalog/grafana/service.yaml [hunk 4]
```diff
@@ -1,15 +1,11 @@
-# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
-# Use of this source code is governed by a BSD-style
-# license that can be found in the LICENSE file.
-
```
Comment (ralgozino) [inline katalog/grafana/service.yaml:4]: please re-add the copyright header

File: katalog/grafana/serviceAccount.yaml [hunk 5]
```diff
@@ -1,7 +1,3 @@
-# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
-# Use of this source code is governed by a BSD-style
-# license that can be found in the LICENSE file.
-
```
Comment (ralgozino) [inline katalog/grafana/serviceAccount.yaml:4]: please re-add the copyright header

File: katalog/grafana/serviceMonitor.yaml [hunk 6]
```diff
@@ -1,15 +1,11 @@
-# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
-# Use of this source code is governed by a BSD-style
-# license that can be found in the LICENSE file.
-
```
Comment (ralgozino) [inline katalog/grafana/serviceMonitor.yaml:4]: please re-add the copyright header

File: katalog/grafana/config.yaml [hunk 7]
```diff
@@ -1,15 +1,11 @@
...
-# Use of this source code is governed by a BSD-style
-# license that can be found in the LICENSE file.
-
```
Comment (ralgozino) [inline katalog/grafana/config.yaml:4]: please re-add the copyright header

File: katalog/grafana/dashboardSources.yaml [hunk 8]
```diff
@@ -1,7 +1,3 @@
-# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
-# Use of this source code is governed by a BSD-style
-# license that can be found in the LICENSE file.
-
```
Comment (ralgozino) [inline katalog/grafana/dashboardSources.yaml:4]: please re-add the copyright header

**Review by ralgozino — CHANGES_REQUESTED** [review]
I was finishing my review, with mostly the same comments that Sam did, he beat me into it :D

**Pull request closed without merging**